src/components/new-safe/steps/Step2/index.tsx

```diff
@@ -95,7 +99,11 @@ const CreateSafeStep2 = ({
             <Divider sx={{ ml: '-52px', mr: '-52px', mb: 4, mt: 3 }} />
             <Typography variant="h4" fontWeight={700} display="inline-flex" alignItems="center" gap={1}>
               Threshold
-              <Tooltip title="TODO: Add tooltip" arrow placement="top">
+              <Tooltip
+                title="The threshold of a Safe specifies, how many owner accounts need to confirm a Safe transaction before it can be executed."
```

Comment:
```suggestion
                title="The threshold of a Safe specifies how many owner accounts need to confirm a Safe transaction before it can be executed."
```